### ISSUE TITLE
Building ffmpeg not working - libsvt_hevc error

### ISSUE THREAD

**Issue by BlackScreen** [m-ab-s/media-autobuild_suite#1369]
```
10:45:56   Running git update for ffmpeg...
10:45:58 ┌ ffmpeg git  ............................... [Recently updated]
10:45:58 ├ Until SVT-AV1 issues a fix, libsvtav1 must be disabled while libsvthevc is enabled....
10:45:58 ├ Changing options to comply to nonfree...
CUDA_PATH environment variable not set or directory does not exist.
10:45:59 Installing clang... done
10:46:04 ├ Compiling static FFmpeg...
10:46:06 ├ Running configure...
10:51:09 ├ Running make...
Likely error:
CC      libavfilter/vf_scale_cuda.ptx.o
CC      libavfilter/vf_thumbnail_cuda.ptx.o
CC      libavfilter/vf_yadif_cuda.ptx.o
AR      libavfilter/libavfilter.a
CC      libavcodec/libsvt_hevc.o
C:/Development/MABS/build/ffmpeg-git/libavcodec/libsvt_hevc.c:24:10: fatal error: EbTime.h: No such file or directory
   24 | #include "EbTime.h"
      |          ^~~~~~~~~~
compilation terminated.
make: *** [/build/ffmpeg-git/ffbuild/common.mak:59: libavcodec/libsvt_hevc.o] Error 1
make failed. Check C:/Development/MABS/build/ffmpeg-git/build-static-64bit/ab-suite.make.log
This is required for other packages, so this script will exit.
11:05:26   Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/z4NC.zip
Copy and paste [logs.zip](https://0x0.st/z4NC.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.

```

**Comment by Helenerineium** [comment]
@moisesmcardona @1480c1 
It seems that [this commit](https://github.com/m-ab-s/media-autobuild_suite/pull/1353/commits/b6bedb925ac7b2f45c187cbeb9d2c87b07177c72) does not make it into the repo during squash and merge.

https://github.com/m-ab-s/media-autobuild_suite/blob/4282bde23104c420a8ea98bfe3fbe2984313cceb/build/media-suite_compile.sh#L1286-L1295

**Comment by BlackScreen** [comment]
Fun fact, after deleting ffmpeg-git in the build directory, building worked. Strange.

**Comment by Helenerineium** [comment]
> the header was removed upstream

It is still in App/ but not required by the patch after [this coomit](https://github.com/OpenVisualCloud/SVT-HEVC/commit/835ec04a4bc9430169657abce5e41cdc6a73f129)

>  the SVT-HEVC patch was outdated

I will suggest to delete the downloaded patch after a successful patch.
Maybe something like `[[ ! -f $patch ]] && rm -f "$patchName"`

**Comment by wiiaboo** [comment]
do_wget already checks for 304, but it might not work for all links (if they're missing handling for If-Modified-Since headers). You could try to apply the local patch first and then retry after overwriting it and only then fail to apply the patch.

**Comment by 1480c1** [comment]
The problem was that the patch applied, but since the patch was outdated, it referenced a header that does not exist anymore

**Comment by wiiaboo** [comment]
Probably fixed by #1378
